### ISSUE TITLE
Font Awesome 4.5 and more

### ISSUE THREAD

**Issue by lp741** [Semantic-Org/Semantic-UI-Docs#165]
just edit **site/elements/icon.overrides.import.less** and add the following code
```
/*******************************
         Site Overrides
*******************************/

/*******************************
            Icons
*******************************/

/* new 4.5.0 icons */

/* Web Content */
i.icon.hashtag:before { content: "\f292"; font-family: FontAwesome}
i.icon.percent:before { content: "\f295"; font-family: FontAwesome}
i.icon.shopping-bag:before { content: "\f290"; font-family: FontAwesome}
i.icon.shopping-basket:before { content: "\f291"; font-family: FontAwesome}

/* Brand Icons */
i.icon.bluetooth:before { content: "\f293"; font-family: FontAwesome}
i.icon.bluetooth-o:before { content: "\f294"; font-family: FontAwesome}
i.icon.codiepie:before { content: "\f284"; font-family: FontAwesome}
i.icon.edge:before { content: "\f282"; font-family: FontAwesome}
i.icon.fort-awesome:before { content: "\f286"; font-family: FontAwesome}
i.icon.mixcloud:before { content: "\f289"; font-family: FontAwesome}
i.icon.modx:before { content: "\f285"; font-family: FontAwesome}
i.icon.product-hunt:before { content: "\f288"; font-family: FontAwesome}
i.icon.reddit-alien:before { content: "\f281"; font-family: FontAwesome}
i.icon.scribd:before { content: "\f28a"; font-family: FontAwesome}
i.icon.usb:before { content: "\f287"; font-family: FontAwesome}

/* Payment Icons */
i.icon.credit-card-alt:before { content: "\f283"; font-family: FontAwesome}

/* Video Player Icons */
i.icon.pause-circle:before { content: "\f28b"; font-family: FontAwesome}
i.icon.pause-circle-o:before { content: "\f28c"; font-family: FontAwesome}
i.icon.stop-circle:before { content: "\f28d"; font-family: FontAwesome}
i.icon.stop-circle-o:before { content: "\f28e"; font-family: FontAwesome}
```

**Comment by IonutBajescu** [comment]
Shouldn't this issue target the main repository? 
https://github.com/Semantic-Org/Semantic-UI

Also, I suggest doing a pull request with this code, it'd have more chances of being merged.

**Comment by jlukic** [comment]
We've merged in the latest Font Awesome in `2.2`
